contribs/gmf/src/directives/displayquerygrid.js

```diff
@@ -274,7 +273,7 @@ gmf.DisplayquerygridController = function($injector, $scope, ngeoQueryResult, ng
 gmf.DisplayquerygridController.prototype.$onInit = function() {
   this.removeEmptyColumns_ = this['removeEmptyColumnsFn'] ? this['removeEmptyColumnsFn']() === true : false;
   this.maxRecenterZoom = this['maxRecenterZoomFn'] ? this['maxRecenterZoomFn']() : undefined;
-  this.mergeTabs_ = this['mergeTabs'] ? this['mergeTabs'] : {};
+  this.mergeTabs = this['mergeTabs'] ? this['mergeTabs'] : {};
```

Comment:
Now you can completely remove this line. If nothing set the property, it will be the property declared in the constructor (so `{}`).

Reply:
true